packages/migrate/src/migrate.ts

```diff
@@ -79,28 +82,28 @@ export async function* migrate(input: MigrateInput): AsyncGenerator<string> {
   const { fileNames: someFiles } = parseJsonConfigFileContent(
     configFile,
     ts.sys,
-    basePath,
+    srcPath,
```

Comment:
`srcPath` here can be a directory path OR a file path. however `parseJsonConfigFileContent` expects only a directory path.

Reply:
That's interesting. 
It sound like we expect srcDir to be a directory... while I also expected to pass path to file as well :

`[srcDir]', 'path to directory containing a package.json`